### PR TITLE
core: add filestore support for dg.py

### PR DESCRIPTION
**Pull request by jschmid1**
$title

-----------------

**Checklist:**
- [x] Added unittests and or functional tests ~#TODO~
- [x] Adapted documentation ~#TODO~ adapted the wiki
~- [ ] Referenced issues or internal bugtracker~
- [ ] Ran integration tests successfully (trigger with "@susebot run teuthology" in a GitHub comment; see the [wiki](https://github.com/SUSE/DeepSea/wiki/Testing) for more information)

### CODE REVIEW

**Comment by jschmid1** [comment]
@susebot run teuthology

**Comment by jschmid1** [comment]
@susebot run teuthology

**Comment by jschmid1** [comment]
retest this please

File: srv/salt/_modules/dg.py [hunk 1]
```diff
@@ -597,6 +603,20 @@ def limit(self) -> int:
...
+        On-disk-format - Filestore/Bluestore
+        """
+        return self.filter_args.get("format", "bluestore")
+
+    @property
+    def journal_size(self) -> int:
+        """
+        Journal size
+        """
+        return self.filter_args.get("journal_size", False)
```
Comment (tserong) [inline srv/salt/_modules/dg.py]: Should the default here be `0`, rather than `False`, to match the specified return type of the function (int)?

**Pull request merged**